package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dash-ag-grid",
-    "version": "31.0.1",
+    "version": "31.2.1",
```

Comment:
I thought our version sync pattern was to match `major.minor` from Ag Grid, and then we reserve `patch` for us. So this bump would be to `31.2.0` and if we need to issue any patch releases without bumping the Ag Grid version _then_ we go to `31.2.1`. But, @alexcjohnson / @BSd3v correct me if I'm wrong.

Reply:
Basically this is a problem because if we match 1:1 then we're blocked from releasing any changes until Ag Grid makes a new release. Ag Grid releases frequently than us so most of the time this should be okay but food for thought.

Reply:
I think that would be the case. We can revert this since you have automated the process of creating a draft. ;)

Reply:
You can also add another commit to this repo just bumping the patch version down to 0; no need to go through the fuss of opening a new PR.

Reply:
Part of my other comment: Basically the changelog should make clear which version (_including the patch_) of Ag Grid we're bundling -- so in our 31.2.0, we bundle 31.2.1: This isn't obvious and should be marked in the changelog.

Reply:
I'll update the release process since this all feels kinda arcane 🙃